src/app_service/service/community/service.nim

```diff
@@ -641,10 +634,20 @@ QtObject:
               )
             )
 
-          # Handle name/description changes
-          if chat.name != prevChat.name or chat.description != prevChat.description or chat.color != prevChat.color or chat.emoji != prevChat.emoji:
-            var updatedChat = findChatById(chat.id, updatedChats)
-            updatedChat.updateMissingFields(chat)
+          if chat.name != prevChat.name or chat.description != prevChat.description or chat.color != prevChat.color or
+              chat.emoji != prevChat.emoji or chat.viewersCanPostReactions != prevChat.viewersCanPostReactions:
+            var updatedChat = chat
+
+            # TODO improve this in https://github.com/status-im/status-desktop/issues/12595
+            # Currently, status-go only sends canPostReactions on app start (getChannelGroups)
+            # so here, we need to imply it. If viewersCanPostReactions is true, then everyone can post reactions
+            # admins can also always post reactions
+            if chat.viewersCanPostReactions == true or
+                (chat.viewersCanPostReactions == false and community.memberRole != MemberRole.None):
+              updatedChat.canPostReactions = true
+            elif chat.viewersCanPostReactions == false and community.memberRole == MemberRole.None:
+              updatedChat.canPostReactions = false
```

Comment:
```suggestion
            if chat.viewersCanPostReactions or
                (!chat.viewersCanPostReactions and community.memberRole != MemberRole.None):
              updatedChat.canPostReactions = true
            elif !chat.viewersCanPostReactions and community.memberRole == MemberRole.None:
              updatedChat.canPostReactions = false
```

Reply:
Yeah, I used those explicit conditions when developing because trying to get the right conditions was a pain so it was easier to understand for me, but then I forgot to simplify haha.

BTW, it's `not` instead of `!` in Nim :wink: